heartcheck-sidekiq.gemspec

```diff
@@ -17,13 +17,15 @@ Gem::Specification.new do |spec|
   spec.test_files = spec.files.grep(/^spec\//)
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'heartcheck', '~> 2.0', '>= 2.0.0'
-  spec.add_dependency 'sidekiq', '>= 2.0.0', '< 6.1.3'
+  spec.add_dependency 'heartcheck', '~> 2.0'
+  spec.add_dependency 'sidekiq', '>= 2.0.0', '<= 6.1.3'
```

Comment:
Do you think that we can leave this dependency version restriction open for newer sidekiq versions?

Actually, we depend on a few methods from sidekiq's public interface in this project (being more objective, we only [depend on `Sidekiq.redis` here](https://github.com/locaweb/heartcheck-sidekiq/blob/master/lib/heartcheck/checks/sidekiq.rb#L12), and since it worked the same, without any kind of changes, since sidekiq early 2.0.0 version up to the newest right now (6.1.3), I guess that this interface is quite stable so that we simply leave it without an "upper maximum version" restriction in the gemspec.

WDYT?